.ci/setenvconfig

```diff
@@ -87,7 +87,7 @@ BUILD_NUMBER=$BUILD_NUMBER
 E2E_PROVIDER=gke
 CLUSTER_NAME=${clusterName}
 TEST_OPTS=-race
-E2E_TEST_LOG_LEVEL=1
+ECK_TEST_LOG_LEVEL=1
```

Comment:
Nit: I think I have a slight preference for `E2E_TEST_LOG_LEVEL` as everthing in this repo is `ECK`

Reply:
I used the `ECK` prefix to avoid any conflicts with environment variables that may already be defined in the user's environment. The reason I didn't go with `E2E` is because the log level change is not specific to E2E test runs. It applies to unit and integration tests too.  That's a minor quibble with semantics though. If you have a strong preference for the `E2E` prefix, I am happy to change it.

Reply:
No that's fine.